.github/workflows/registry-2.0.yml

```diff
@@ -120,8 +120,7 @@ jobs:
 
       - name: Generate helm values
         run: |
-          INGRESS_IP=$(kubectl get svc nginx-ingress-controller -n nginx-ingress \
-          -o json | jq -r ".status.loadBalancer.ingress[].ip")
+          INGRESS_IP="$(kubectl get svc --all-namespaces -l app=nginx-ingress | awk '/LoadBalancer/ { print $5 }')"
```

Comment:
So why don't you install ingress into `nginx-ingress` namespace? (sorry I did not check other parts of the code..)

What about `kubectl get svc --all-namespaces -l app=nginx-ingress -l component=controller -o jsonpath="{.items[].status.loadBalancer.ingress[].ip}"` ?

Reply:
> So why don't you install ingress into `nginx-ingress` namespace? (sorry I did not check other parts of the code..)
To install additional services on the cluster I am using the same logic as in the aks/eks workflwos where it uses the upstream nginx-ingress. However for CaaSP we use th suse nginx-ingress in that case I need a different name to differentiate it, that's why I install it on the nginx-ingress-suse namespace.
> What about `kubectl get svc --all-namespaces -l app=nginx-ingress -l component=controller -o jsonpath="{.items[].status.loadBalancer.ingress[].ip}"` ?
Well, I believe there are several ways to achieve the same result, is there any advantage on the one you mentioned?

Reply:
Not really, I just like to achieve the result with `kubectl` alone, without the need to grep/awk the results